index.js

```diff
@@ -16,6 +16,14 @@ function promisifyOwnProperties(obj) {
 }
 
 module.exports = obj => {
+  if (!obj) {
+    return obj;
+  }
+
+  if (typeof obj !== 'function' && (typeof obj !== 'object' || Array.isArray(obj))) {
```

Comment:
(pointer back to https://github.com/SimenB/util.promisify-all/pull/4#discussion_r119971126 re arrays)